test/test_utils/eks.py

```diff
@@ -279,19 +279,25 @@ def create_eks_cluster(eks_cluster_name, processor_type, num_nodes,
     eksctl_create_cluster_command += " --auto-kubeconfig "
     run(eksctl_create_cluster_command)
 
+    eks_write_kubeconfig(eks_cluster_name, "us-west-2")
+
+    run(
```

Comment:
I know right now we're spinning up GPU clusters, but will this break with a CPU cluster?

Reply:
I assume you're referring to the NVIDIA device plugin change.
We've been applying the device plugin for a while now in the code with all clusters that are created. So I don't think it should change anything significantly to do this here.